### ISSUE TITLE
[Accessibility] Floating action button not accessible by keyboard interaction

### ISSUE THREAD

**Issue by gselderslaghs** [materializecss/materialize#540]
### Before submitting...

- [X] I have searched for [duplicate or closed issues](https://github.com/materializecss/materialize/issues?utf8=%E2%9C%93&q=is%3Aissue).
- [X] I have read the [CONTRIBUTING document](https://github.com/materializecss/materialize/blob/master/CONTRIBUTING.md) and my issue is following the guidelines.
- [X] I have read the template completely before filling it in.

### Context

When using the Floating action button component, the element does not seem to be intractable by keyboard

### Current Behavior

Button is not tabbable/focusable by keyboard interaction

[Codepen example](https://codepen.io/Geert-Selderslaghs/pen/zxOqovQ)

### Expected behavior

Component should be accessible by mouse and keyboard input

### Possible Solutions or Causes

Implement the necessary functional attributes on initialization in order to make button tabbable/focusable by keyboard interaction

### Your Environment

* Version used: 2.1.1

**Comment by gselderslaghs** [comment]
Closing this issue as fixed in release v2.2.2
